diff --git a/istioctl/cmd/root.go b/istioctl/cmd/root.go
index 9a5da0f290..f4f05cda44 100644
--- a/istioctl/cmd/root.go
+++ b/istioctl/cmd/root.go
@@ -126,7 +126,7 @@ func init() {
 // GetRootCmd returns the root of the cobra command-tree.
 func GetRootCmd(args []string) *cobra.Command {
 	rootCmd := &cobra.Command{
-		Use:               "istioctl",
+		Use:               "istioctl-1.8.5",
 		Short:             "Istio control interface.",
 		SilenceUsage:      true,
 		DisableAutoGenTag: true,
