diff --git a/istioctl/cmd/root.go b/istioctl/cmd/root.go
index deaf3b064a..e607373e12 100644
--- a/istioctl/cmd/root.go
+++ b/istioctl/cmd/root.go
@@ -131,7 +131,7 @@ func init() {
 // GetRootCmd returns the root of the cobra command-tree.
 func GetRootCmd(args []string) *cobra.Command {
 	rootCmd := &cobra.Command{
-		Use:               "istioctl",
+		Use:               "istioctl-1.10.3",
 		Short:             "Istio control interface.",
 		SilenceUsage:      true,
 		DisableAutoGenTag: true,
